utils-simd/ppv-lite86/src/generic.rs

```diff
@@ -4,9 +4,10 @@ use crate::soft::{x2, x4};
 use crate::types::*;
 use core::ops::*;
 use zerocopy::{AsBytes, FromBytes};
+use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
```

Comment:
This results in some duplicate definitions which I observed with the `aarch64-unknown-linux-musl` target.

```
error[E0252]: the name `AsBytes` is defined multiple times
 --> $HOME/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ppv-lite86-0.2.19/src/generic.rs:7:23
  |
6 | use zerocopy::{AsBytes, FromBytes};
  |                ------- previous import of the macro `AsBytes` here
7 | use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
  |                       ^^^^^^^--
  |                       |
  |                       `AsBytes` reimported here
  |                       help: remove unnecessary import
  |
  = note: `AsBytes` must be defined only once in the macro namespace of this module

error[E0252]: the name `FromBytes` is defined multiple times
 --> $HOME/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ppv-lite86-0.2.19/src/generic.rs:7:32
  |
6 | use zerocopy::{AsBytes, FromBytes};
  |                         --------- previous import of the macro `FromBytes` here
7 | use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
  |                                ^^^^^^^^^--
  |                                |
  |                                `FromBytes` reimported here
  |                                help: remove unnecessary import
  |
  = note: `FromBytes` must be defined only once in the macro namespace of this module
```

Looking at the other changes it seems like the intention was to remove the `zerocopy::` ones?

Reply:
`i586-unknown-linux-gnu` also shows the issue and is probably easier to test with.

The issue needs the `zeroconf/derive` feature to be enabled (another dep in my dep tree does so).

Removing the duplicate line would be a quick fix, but perhaps the right thing to do is enable the feature and use that instead of importing both zeroconf and zeroconf-derive?

Reply:
Many thanks for the explanation, I never would have figured that out!